prospector/core/prospector.py

```diff
@@ -157,7 +158,14 @@ def prospector(  # noqa: C901
                     exc_info=get_level() < logging.WARNING,
                 )
                 if use_backend == "always":
-                    print("Backend not reachable: aborting")
+                    if backend_address == "http://localhost:8000" and os.path.exists(
```

Comment:
please remove this "magic string" (=no string constants should appear in the code; refer to a variable/constant defined at the top of the module instead)

Reply:
also, comparing this with the string as such is quite brittle (how about http://localhost:8000/ ?)

Reply:
Should I change this (to compare to constant and be less brittle) or remove it? I added this check because I was stuck on this problem for so long, but I'm not sure if this should be part of the code in general. 
Instead I could add some more explanation for the `backend` field in `config-sample.yaml`?